src/pages/public_registration/PublicPageInfoEntry.tsx

```diff
@@ -9,7 +9,15 @@ interface PublicPageInfoEntryProps {
 
 export const PublicPageInfoEntry = ({ title, content, ...rest }: PublicPageInfoEntryProps) => (
   <Box sx={{ display: 'grid', gridTemplateColumns: 'auto 1fr', gap: '0.25rem' }}>
-    <Typography sx={{ textWrap: 'nowrap', fontWeight: 700 }}>{title}:</Typography>
-    {typeof content === 'string' ? <Typography {...rest}>{content}</Typography> : <div {...rest}>{content}</div>}
+    <Typography component="dt" sx={{ textWrap: 'nowrap', fontWeight: 700 }}>
+      {title}:
+    </Typography>
+    {typeof content === 'string' ? (
+      <Typography component="dd" {...rest}>
+        {content}
+      </Typography>
+    ) : (
+      <dd {...rest}>{content}</dd>
```

Comment:
Dette er vel typisk en liste over flere ting, og alle burde være hver sin <dd> i stedet for at de wrappes i en felles? Lurer derfor på om dette heller skulle vært en div/span og at vi må sende inn `component="dd"` hvor det trengs for content